BE/src/index.ts

```diff
@@ -135,9 +136,12 @@ const runServer = async () => {
 async function Execute() {
     GenerateManagementToken();
     await runServer();
-    //await Auth0_Database.RESET();
-    //const allUsers = await Auth0_Database.SELECT_STAR_FROM_ESSENTIAL();
-    //console.log(allUsers);
+
+    const packages = ["axios", "mongoose"];
+    const packageManager = "npm"; // or 'yarn'
+    const totalSize = calculateCumulativeSize(packages, packageManager);
+
+    console.log(`Total size cost for packages [${packages.join(", ")}]: ${totalSize / 1024} KB`);
```

Comment:
please comment out these lines, then it's good to go

Reply:
ok